src-built-in/components/UIAPI/reducers/linker.spec.ts

```diff
@@ -0,0 +1,168 @@
+/**
+ * Mody: Prior this, there was no tests folder and
+ * there wasn't any configuration for mocha or any other
+ * testing framework, placing spec files in the same
+ * directory for now (personal pref) I can move them
+ * to another folder later on if requested to do so.
+ */
+import { linker as reducer, initialState } from './linker';
+import { loop, Cmd } from 'redux-loop';
+import { linkChannel, initializeLinker, cleanUp, fitDOM } from '../effects/linker';
+import { toggleSuccess, toggleFailure, initSuccess } from '../actions/linkerActions';
+import * as actions from '../actions/linkerActions';
+import { assert } from 'chai';
+import 'mocha';
```

Comment:
Usually you don't need to import mocha, is this to get rid of some the ltslint warnings?

Reply:
Yes, otherwise I would have the red squiggly...